monai/transforms/inverse.py

```diff
@@ -122,7 +122,8 @@ def push_transform(self, data, *args, **kwargs):
             if do_transform:
                 xform = data.pending_operations.pop()
                 xform.update(transform_info)
-                meta_obj = self.push_transform(data, transform_info=xform, lazy_evaluation=lazy_eval)
+                extra = xform.copy()
+                meta_obj = self.push_transform(data, transform_info=xform, lazy_evaluation=lazy_eval, extra_info=extra)
```

Comment:
Hi @yiheng-wang-nv,
this branch uses:
```py
extra = xform.copy()
xform.update(transform_info)
```
but the prototype branch uses:
```py
xform.update(transform_info)
extra = xform.copy()
```
they are different, is this a typo or a bug fix?

Reply:
Thanks @wyli for correction. It was a typo